config.yaml

```diff
@@ -30,3 +30,14 @@ options:
     description: |
       The service binds to all network interfaces if true. The service binds
       only to the first found bind address of each relation if false
+  prometheus_datasource:
```

Comment:
I think this config parameter shouldn't exist, but rather infer the "appname" from the existing relation between the prometheus Juju app and the etcd app.